styles/Home.module.css

```diff
@@ -12,6 +12,16 @@
   align-items: center;
 }
 
+.contest_container {
+  min-height: 100vh;
```

Comment:
We shouldn't be using custom css, just use whatever tailwind provides

Reply:
not beeing used anymore